etl-backup/upload-etl.sh

```diff
@@ -52,7 +52,7 @@ kubectl cp -c cost-model $etlFile $namespace/$podName:/var/configs/kubecost-etl.
 # Exec into the pod and replace the ETL
 echo "Execing into the pod and replacing $etlDir "
 set -e
-kubectl exec -n $namespace pod/$podName -- ash -c " \
+kubectl exec -n $namespace pod/$podName -- bash -c " \
```

Comment:
Well, actually, I think our ubuntu container might use `ash` as a shell... I think we would be best-served with `sh` for portability